src/components/pages/BeneficiosGerenciar.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import apiBeneficio from "../../services/apiBeneficio";
+import CardBenefit from "../components/CardBenefit";
+
+class BeneficiosGerenciar extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      benefits: null
+    };
+    //this.getBenefits();
+    this.colors = ["#6FB1DE", "#438ABB", "#226D9F", "#0F4C75"];
+  }
+
+  async getBenefits(){
+    const response = await apiBeneficio.get("benefits");
+    this.setState({benefits: response.data});
+  }
+
+  componentDidMount(){
+    this.getBenefits();
+  }
+
+  deleteBenefits = async (benefit_id) =>{
+    if(window.confirm("Tem certeza que quer excluir o benefício?")){
+      try{
+        await apiBeneficio.delete(`/benefits/${benefit_id}`);
```

Comment:
Codacy found an issue: [Identifier 'benefit_id' is not in camel case.](https://app.codacy.com/gh/fga-eps-mds/2020.1-VC_Gestor-FrontEnd/pullRequest?prid=6353377)